packages/admin-next/dashboard/src/hooks/table/query/use-return-reason-table-query.tsx

```diff
@@ -0,0 +1,32 @@
+import { HttpTypes } from "@medusajs/types"
+import { useQueryParams } from "../../use-query-params"
+
+type UseReturnReasonTableQueryProps = {
+  prefix?: string
+  pageSize?: number
+}
+
+export const useReturnReasonTableQuery = ({
+  prefix,
+  pageSize = 20,
+}: UseReturnReasonTableQueryProps) => {
+  const queryObject = useQueryParams(
+    ["offset", "q", "order", "created_at", "updated_at"],
+    prefix
+  )
+
+  const { offset, q, order, created_at, updated_at } = queryObject
+  const searchParams: HttpTypes.AdminReturnReasonListParams = {
+    limit: pageSize,
+    offset: offset ? Number(offset) : 0,
+    order,
+    created_at: created_at ? JSON.parse(created_at) : undefined,
```

Comment:
Why are we parsing the dates here as JSON.parse? Isn't it better to use `Date.parse`?